app/questionnaire/placeholder_parser.py

```diff
@@ -140,6 +163,62 @@ def _resolve_value_source_list(
                 values.append(value)
         return values
 
+    def _get_routing_path_block_ids(
+        self, sections_to_ignore: list | None = None
+    ) -> dict[str, list[str]]:
+        if self._location and self._router:
+            return get_block_ids_for_calculated_summary_dependencies(
```

Comment:
This doesn't cover that we might not pass then even when we need it. Can we not always have these?. If not, raise an error in the methods that uses it. At the moment, it would silently pass.

Reply:
This is still an issue but we are okay for now.